pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -19,31 +19,53 @@
 
 public class PythonVersion implements Serializable {
 
-    private final String version;
+    private final String major;
+    private final String minor;
+    private final String patch;
 
     public PythonVersion(String version) {
-        this.version = version;
+        String[] split = version.split("\\.");
+        this.major = split[0];
+        this.minor = split[1];
+        if (split.length < 3) {
+            this.patch = "0";
```

Comment:
@zvezdan let me know if this is a bad assumption. CPython has been versioned with major/minor for a long time, so we might want to set this to `null` instead of `0`. Not sure.